### ISSUE TITLE
Octomaps not visualized when inside a gap of a mesh 

### ISSUE THREAD

**Issue by marcoojer** [ros-planning/moveit#2280]
Hi, I'm loading a pointCloud and trying to visualize its octomap. Usually it works as expected, but when a part of the octomap is in the gap of a non-convex mesh it is not visualized. Here I show some images to illustrate this proble.

Working case:
![image](https://user-images.githubusercontent.com/22325546/91432700-652b5880-e862-11ea-8ef5-c41025fceeec.png)

Not-Working case:
![image](https://user-images.githubusercontent.com/22325546/91432943-c6532c00-e862-11ea-9545-4fcba1429806.png)
It can be seen that the part that is inside the cell is not visualized although it does not appear to be in collision with the mesh.
In both images I show the robot collision part of the urdf. 
It seems that the when creating the octomap from the point cloud is using the convex hull of the mesh. Anyone has any idea about how to solve that?

**Comment by simonschmeisser** [comment]
Yes, it uses the convex hull for 'self filtering'. You might want to check if all updaters act the same. (point cloud/depth map)

You could add some complex decomposition to the updater or split your meshes by hand. Please keep us up to date on how that goes and provide a PR.
